spec/factories.rb

```diff
@@ -119,14 +119,14 @@
     transient do
       title "An Unpublished Point"
       body "Some Body Text"
-      summary "A summary"
+      description "A summary"
```

Comment:
In a perfect world we'd have a reason to use this factory because we're not republishing the standard if we unpublish a point. But as there is no plan to change the number of points anytime soon and we'll get the unpublishing thing for free when we move to the links hash.. we get away with it.